packages/fluent/scss/drawer/_layout.scss

```diff
@@ -0,0 +1,255 @@
+@use "../core/color-system" as *;
+@use "_variables.scss" as *;
+
+@mixin kendo-drawer--layout() {
+
+    .k-drawer-container {
+        display: flex;
+        flex-flow: row nowrap;
+        align-items: flex-start;
+    }
+    kendo-drawer.k-drawer,
+    kendo-drawer .k-drawer-wrapper {
+        transition: none;
+    }
+    .k-drawer {
+        height: 100%;
+        max-width: 100%;
+        border-width: 0;
+        border-style: solid;
+        box-sizing: border-box;
+        outline: 0;
+        font-family: var( --kendo-drawer-font-family, #{$kendo-drawer-font-family} );
+        font-size: var( --kendo-drawer-font-size, #{$kendo-drawer-font-size} );
+        line-height: var( --kendo-drawer-line-height, #{$kendo-drawer-line-height} );
+        display: flex;
+        flex-direction: column;
+        flex: 0 0 auto;
+        transition: all 300ms ease-in-out; // sass-lint:disable-line no-transition-all
```

Comment:
you can use --kendo-transition here ?

Reply:
When I changed it to `--kendo-transition` the opening animation of Drawer disappeared, at least in the visual tests (when refreshing the page).
Therefore I'm leaving it as-is.